google-ads/src/main/java/com/google/ads/googleads/lib/catalog/Version.java

```diff
@@ -50,6 +50,14 @@
         versionName.matches("v[0-9]+"), "Invalid version name: %", versionName);
   }
 
+  /**
+   * Returns the version name, e.g. "v1".
+   * @return the version name
+   */
+  public String getVersionName() {
```

Comment:
Could you elaborate on the motivation behind this change?

Reply:
Accessing the name (i.e. the number) of the latest version programmatically may be useful for client applications that want to use reflection to instantiate classes like resources or enums, like an entity class generator that generates a class for each resource and wants to use the appropriate class for the enum fields.